fvm/state/state_test.go

```diff
@@ -6,48 +6,109 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/onflow/flow-go/fvm/state"
+	"github.com/onflow/flow-go/fvm/utils"
 )
 
-func TestState_DraftFunctionality(t *testing.T) {
-	ledger := state.NewMapLedger()
-	st := state.NewState(ledger)
+func TestState_ChildMergeFunctionality(t *testing.T) {
```

Comment:
these tests would be great if supported by documentation. In general, describing what each tries to evaluate.

Reply:
fixed.